docs/integrations/microsoft-azure/opentelemetry/active-directory-json-opentelemetry.md

```diff
@@ -0,0 +1,140 @@
+---
+id: active-directory-json-opentelemetry
+title: Active Directory JSON - OpenTelemetry Collector
+sidebar_label: Active Directory JSON - OTel Collector
+description: Learn about the Sumo Logic OpenTelemetry App for Active Directory JSON.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/windows.png')} alt="Thumbnail icon" width="40"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The [Active Directory](https://learn.microsoft.com/en-us/windows-server/identity/ad-ds/get-started/virtual-dc/active-directory-domain-services-overview) JSON App helps you monitor your Windows Active Directory deployment by analyzing Active Directory logs in the JSON based event log format. The app includes predefined searches and dashboards that provide user activity into your environment for real-time analysis of overall usage.
+
+We recommend using the Active Directory JSON App in combination with the Windows JSON App.Active Directory logs are sent to Sumo Logic through OpenTelemetry [windowseventlogreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/windowseventlogreceiver).
+
+<img src={useBaseUrl('https://https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Active-Directory-OpenTelemetry/Active-Directory-Schematics.png')} alt="Schematics" />
+
+## Fields creation in Sumo Logic for Active Directory
+
+Following are the [fields](https://help.sumologic.com/docs/manage/fields/) which will be created as part of Active Directory App install if not already present.
+
+**`sumo.datasource`** - Has fixed value of **activeDirectory**
+
+### Event Logs Used by Active Directory App
+
+This section provides instructions for configuring log collection for Active Directory running on a non-Kubernetes environment for the Sumo Logic App for Active Directory.
+
+#### Log Types
+
+Standard Windows event channels include:
+
+-   **Security**
+-   **System**
+-   **Application**
+
+## Collection Configuration & App installation
+
+As part of the setting up the collection process and app installation, user can select the App from App Catalog and click on Install App. Please follow the steps below:
```

Comment:
```suggestion
As part of setting up the collection process and app installation, you can select the App from App Catalog and click on Install App. Follow the steps below.
```

Reply:
There is no "Install App" button anymore. You should update the installation steps. It's "Add Integration" button.

Reply:
@JainM6 Great catch!! Thank you so much.

Reply:
I checked with the S3M team. It looks like they're reverting back to **Install App** in the new UI.

<img width="497" alt="Screenshot 2023-03-25 at 9 59 20 AM" src="https://user-images.githubusercontent.com/56411016/227731459-8e7f662f-b22b-472e-9d16-d19c917264a6.png">